src/Planners/include/BipedalLocomotion/Planners/UnicycleTrajectoryPlanner.h

```diff
@@ -194,45 +210,47 @@ class BipedalLocomotion::Planners::UnicycleTrajectoryPlanner final
+     * | `saturationFactors`       | list of 2 doubles |        -          |   (0.7 0.7)     |  Linear and Angular velocity conservative factors                                    |
+     * | `leftYawDeltaInDeg`       |     double        |        0.0        |       -         | Offset for the left foot rotation around the z axis                                  |
+     * | `rightYawDeltaInDeg`      |     double        |        0.0        |       -         | Offset for the right foot rotation around the z axis                                 |
+     * |      `swingLeft`          |      bool         |       false       |       -         |     Perform the first step with the left foot                                        |
+     * | `startAlwaysSameFoot`     |      bool         |       false       |       -         |       Restart with the default foot if still                                         |
+     * |     `terminalStep`        |      bool         |       true        |       -         |   Add a terminal step at the end of the horizon                                      |
+     * |     `mergePointRatios`    | list of 2 doubles |         -         |   (0.4 0.4)     | The ratios of the DS phase in which it is present a merge point                      |
+     * | `switchOverSwingRatio`    |     double        |        0.2        |       -         | The ratio between single and double support phases                                   |
+     * | `lastStepSwitchTime`      |     double        |        0.3        |       -         |       Time duration of double support phase in final step                            |
+     * | `isPauseActive`           |      bool         |       true        |       -         |    If true, the planner can pause, instead of make tiny steps.                       |
+     * | `comHeight`               |     double        |        0.70       |       -         |    CoM height in double support phase                                                |
+     * | `comHeightDelta`          |     double        |        0.01       |       -         |    Delta to add to CoM heinght in Single support phases                              |
+     * | `leftZMPDelta`            | list of 2 doubles |         -         |   (0.0  0.0)    | Local ZMP reference: delta wrt center frame of the foot                              |
+     * | `rightZMPDelta`           | list of 2 doubles |         -         |   (0.0  0.0)    | Local ZMP reference: delta wrt center frame of the foot                              |
+     * | `leftZMPInitSwitchDelta`  | list of 2 doubles |    (0.0  0.0)     |   (0.0  0.0)    | Local ZMP reference: delta wrt center frame when the switch to the other foot begin  |
+     * | `rightZMPInitSwitchDelta` | list of 2 doubles |    (0.0  0.0)     |   (0.0  0.0)    | Local ZMP reference: delta wrt center frame when the switch to the other foot begin  |
```

Comment:
I would suggest specifying this is used only for the ZMP generator

Reply:
Sure, done with [52e42e3](https://github.com/ami-iit/bipedal-locomotion-framework/pull/916/commits/52e42e31f0dc10775f1160cf3707d907dce23f4e)